Leanplum/Common/LPApiConstants.h

```diff
@@ -55,8 +63,13 @@
 @property(assign, nonatomic) BOOL networkActivityIndicatorEnabled;
 @property(assign, nonatomic) BOOL isLocationCollectionEnabled;
 @property(assign, nonatomic) BOOL isInboxImagePrefetchingEnabled;
+// Counts how many user code blocks we're inside, to silence exceptions.
+// This is used by LP_BEGIN_USER_CODE and LP_END_USER_CODE, which are threadsafe.
+@property(assign, nonatomic) int userCodeBlocks;
 
 + (LPApiConstants *)sharedState;
++ (NSString *)getApiMethod:(LPApiMethod)apiMethodEnum;
```

Comment:
Instead of having this in `ApiConstants`, maybe we can move this to its own file `LPApiMethods`